config/manager/manager.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         - /manager
         args:
         - --enable-leader-election
-        image: quay.io/fpaoline/metallboperator:latest
+        image: controller # Should be controller, this is the name that kustomize looks for and edits on deploy
```

Comment:
How will this behave when we want to use config just to deploy?

Reply:
applying this yaml directly wouldn't work I guess, however make deploy expects that image to be named controller (and replaces it with IMG var in Makefile)